src/test/java/com/sphy/hotelmanagementapplication/service/RoomServiceTest.java

```diff
@@ -192,7 +193,7 @@ void disableRoom() {
 		Room room = new Room();
 		room.setId(1L);
 		room.setName("room");
-		room.setDisabled(true);
```

Comment:
this should stay false, for actually checking if method that we test it makes the value true